google-cloud-storage/src/test/java/com/google/cloud/storage/it/ITGrpcTest.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.storage.it;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import com.google.cloud.NoCredentials;
+import com.google.cloud.storage.Bucket;
+import com.google.cloud.storage.BucketInfo;
+import com.google.cloud.storage.Storage;
+import com.google.cloud.storage.StorageFixture;
+import com.google.cloud.storage.StorageOptions;
+import com.google.cloud.storage.conformance.retry.TestBench;
+import com.google.cloud.storage.testing.RemoteStorageHelper;
+import java.util.logging.Logger;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
+
+public final class ITGrpcTest {
+  private static final Logger LOGGER = Logger.getLogger(ITGrpcTest.class.getName());
+  private static final String TESTBENCH_GRPC_URI = "http://localhost:9001";
+
+  @ClassRule
+  public static final TestBench TEST_BENCH =
+      TestBench.newBuilder().setGRPCBaseUri(TESTBENCH_GRPC_URI).build();
+
+  @Rule
+  public final StorageFixture storageFixture =
+      StorageFixture.from(
+          () ->
+              StorageOptions.grpc()
+                  .setHost(TESTBENCH_GRPC_URI)
```

Comment:
This can be `TEST_BENCH.getGRPCBaseUri()`. The actual reading of this value will happen after TestBench has already booted.